### ISSUE TITLE
Lookup from TypeScript file only searches JavaScript

### ISSUE THREAD

**Issue by medmunds** [gdelmas/IntelliJDashPlugin#79]
Searches for TypeScript features in a .ts file seem to end up in the JavaScript section of Dash.

To reproduce:

1. Create file foo.ts (in, say, WebStorm):
```typescript
interface Foo {
  bar: string;
}
```

2. Position the caret in `interface` and press Cmd-Shift-D.

Results: searches for "javascript: interface". (And I have Node docset installed, so it finds a match there.)
  <img width="472" alt="screen shot 2019-03-07 at 2 14 32 pm" src="https://user-images.githubusercontent.com/639984/53993243-b03c3380-40e3-11e9-9bba-3a81a19854f5.png">

Expected: should search for "typescript: interface"
  <img width="423" alt="screen shot 2019-03-07 at 2 15 04 pm" src="https://user-images.githubusercontent.com/639984/53993259-baf6c880-40e3-11e9-9750-04f3f0a28876.png">

Dash plugin v3.3
WebStorm 2018.3.5
Dash v4.6.2

**Comment by gdelmas** [comment]
you are right. when searching in typescript files it should prioritize the results from the typescript docset, yet still search in all the docsets defined for javascript. as the javascript commands and api still apply.

i'll add this.

**Comment by steven130169** [comment]
same issue
